scripts/apps/search/components/FetchedDesksInfo.tsx

```diff
@@ -71,22 +71,27 @@ export class FetchedDesksInfo extends React.Component<any, any> {
             const desk = this.state.desks[0];
             const name = this.formatDeskName(desk.desk.name);
 
-            items.push(React.createElement('dd', {key: 'dd1'}, desk.isUserDeskMember ?
-                React.createElement('a', {onClick: this.openDesk(desk)}, name) :
-                React.createElement('span', {className: 'container'}, name),
-            ));
+            items.push(
```

Comment:
It would be cleaner to "interpolate" inside JSX instead of creating a separate array holding JSX items. Since there is only one item, keys wouldn't be needed either.

```
return (
            <div>
                <dl>
                    {
                        desk.isUserDeskMember
                            ? (<a onClick={this.openDesk(desk)}>{name}</a>)
                            : (<span className="container">{name}</span>)
                    }
                </dl>
            </div>
        );
```